src/arviz_plots/plots/essplot.py

```diff
@@ -0,0 +1,251 @@
+        Variable in sample stats to use as rug mask. Must be a boolean variable.
+    n_points : int, default 20
+        Number of points for which to plot their quantile/local ess or number of subsets
+        in the evolution plot.
+    extra_methods : bool, default False
+        Plot mean and sd ESS as horizontal lines. Not taken into account if ``kind = 'evolution'``.
+    min_ess : int, default 400
+        Minimum number of ESS desired. If ``relative=True`` the line is plotted at
+        ``min_ess / n_samples`` for local and quantile kinds and as a curve following
+        the ``min_ess / n`` dependency in evolution kind.
+    plot_collection : PlotCollection, optional
+    backend : {"matplotlib", "bokeh"}, optional
+    labeller : labeller, optional
+    aes_map : mapping of {str : sequence of str or False}, optional
+        Mapping of artists to aesthetics that should use their mapping in `plot_collection`
+        when plotted. Valid keys are the same as for `plot_kwargs`.
+
+    plot_kwargs : mapping of {str : mapping or False}, optional
+        Valid keys are:
+
+        * One of "local", "quantile", "evolution", matching the `kind` argument.
+            * "local" -> passed to :func:`~arviz_plots.visuals.scatter_xy`
+            * "quantile" -> passed to :func:`~arviz_plots.visuals.line_xy`
+            * "evolution" -> passed to :func:`~arviz_plots.visuals.line_xy`
+
+        * divergence -> passed to :func:`~.visuals.trace_rug`
+        * title -> passed to :func:`~arviz_plots.visuals.labelled_title`
+
+    stats_kwargs : mapping, optional
+        Valid keys are:
+
+        * ess -> passed to ess
+
+    pc_kwargs : mapping
+        Passed to :class:`arviz_plots.PlotCollection.wrap`
+
+    Returns
+    -------
+    PlotCollection
+    """
+    # initial defaults
+    if sample_dims is None:
+        sample_dims = rcParams["data.sample_dims"]
+    if isinstance(sample_dims, str):
+        sample_dims = [sample_dims]
+
+    # mutable inputs
+    if plot_kwargs is None:
+        plot_kwargs = {}
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+
+    if stats_kwargs is None:
+        stats_kwargs = {}
+
+    # processing dt/group/coords/filtering
+    distribution = process_group_variables_coords(
+        dt, group=group, var_names=var_names, filter_vars=filter_vars, coords=coords
+    )
+
+    # set plot collection initialization defaults if it doesnt exist
+    if plot_collection is None:
+        if backend is None:
+            backend = rcParams["plot.backend"]
+        pc_kwargs.setdefault("col_wrap", 5)
+        pc_kwargs.setdefault(
+            "cols",
+            ["__variable__"]
+            + [dim for dim in distribution.dims if dim not in {"model"}.union(sample_dims)],
+        )
+        if "model" in distribution:
+            pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+            pc_kwargs["aes"].setdefault("color", ["model"])
```

Comment:
```suggestion
        if "model" in distribution:
            pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
            pc_kwargs["aes"].setdefault("color", ["model"])
```

For this plot I think it would look better to also add `x` to the default aesthetic mappings when there are multiple models.